src/coreclr/src/jit/rangecheck.cpp

```diff
@@ -627,6 +670,31 @@ void RangeCheck::MergeEdgeAssertions(GenTreeLclVarCommon* lcl, ASSERT_VALARG_TP
             limit   = Limit(Limit::keConstant, info.constVal);
             cmpOper = (genTreeOps)info.cmpOper;
         }
+        // Current assertion is of the form i == 100
```

Comment:
Comment should read:

 // Is the current assertion a valid constant int32 assertion?

Reply:
This is the pre-existing style for this else if chain. I can change them over, but I personally like having the form there.